circle.yml

```diff
@@ -5,13 +5,15 @@ machine:
 dependencies:
   pre:
     - gem install xcpretty --no-rdoc --no-ri --no-document --quiet
+    - Scripts/install_swiftlint.sh
     - if [[ $CIRCLE_BRANCH == master ]]; then pod setup; fi
   cache_directories:
     -  "~/.cocoapods/repos/master"
 
 test:
   override:
+    - rake lint:code
     - rake xcode:test
     - rake spm:test
   post:
-    - if [[ $CIRCLE_BRANCH == master ]]; then rake lint; fi
+    - if [[ $CIRCLE_BRANCH == master ]]; then rake lint:pod; fi
```

Comment:
Other alternative: `[[ $CIRCLE_BRANCH == master ]] && rake lint:post` (which would actually be `[[ $CIRCLE_BRANCH == master ]] && pod lib lint --quick` if we take the pod lint out of the Rakefile).
I have no real preference, just saying in case you prefer `&&` over `if; then; fi`